larq/quantized_variable_test.py

```diff
@@ -299,3 +300,15 @@ def f():
             assert evaluate(x) == -2.0
     else:
         assert evaluate(x) == 0.0
+
+
+@pytest.mark.skipif(
+    version.parse(tf.__version__) < version.parse("2"), reason="Requires TensorFlow 2",
+)
```

Comment:
I guess we could make this work in TF 1.x as well, but ignore the failure for now.